templates/taxi/driver_detail.html

```diff
@@ -22,5 +22,7 @@
     {% empty %}
       <p>No cars!</p>
     {% endfor %}
+  <a style="float: left" href="{% url "taxi:driver-update" pk=driver.id %}" class="btn btn-secondary">Update</a>
+  <a style="float: left" href="{% url "taxi:driver-delete" pk=driver.id %} " class="btn btn-danger">Delete</a>
```

Comment:
The URL name `taxi:driver-delete` should match the convention used in the `urls.py` file, where it is named `driver-delete`. Ensure consistency in URL naming to avoid errors.